internal/definitions/definition_handler_contracts_test.go

```diff
@@ -106,7 +106,12 @@ func TestHandleFFIBroadcastOk(t *testing.T) {
 	mbi.On("InsertEvent", mock.Anything, mock.Anything).Return(nil)
 	mcm := dh.contracts.(*contractmocks.Manager)
 	mcm.On("ValidateFFIAndSetPathnames", mock.Anything, mock.Anything).Return(nil)
-
+	rag := mbi.On("RunAsGroup", mock.Anything, mock.Anything, mock.Anything)
```

Comment:
A bunch of this will need to be ripped back out if RunAsGroup is removed from the definition handler (sorry)